src/main/java/org/commcare/formplayer/services/MediaMetaDataService.java

```diff
@@ -20,6 +20,14 @@ public int purge() {
         return mediaMetaDataRepo.deleteMetaDataWithoutFormSessionId();
     }
 
+    public MediaMetadataRecord findById(String id) {
+        try {
+            return mediaMetaDataRepo.findById(id).get();
+        } catch (MediaMetaDataNotFoundException e) {
```

Comment:
this exception won't get thrown by itself from `mediaMetaDataRepo.findById(id)` but we will need to manually check if we got any result from `mediaMetaDataRepo.findById` and throw an exception. Look [here](https://github.com/dimagi/formplayer/blob/master/src/main/java/org/commcare/formplayer/services/FormSessionService.java#L58) for a similar example with form session repo.